var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -155,7 +155,7 @@ class Gdal(AutotoolsPackage):
     depends_on('python@2.0:', type=('build', 'link', 'run'), when='@3.2:+python')
     depends_on('python', type=('build', 'link', 'run'), when='+python')
     # swig/python/setup.py
-    depends_on('py-setuptools', type='build', when='+python')
+    depends_on('py-setuptools@:57', type='build', when='+python')
```

Comment:
I just built GDAL 3.4.0 with py-setuptools 59.4.0, are you sure they still use 2to3?

Reply:
Gdal removed 2to3 from their package setup in version 3.1.0. I'll specify that, thanks

Reply:
It looks like 2to3 was actually removed in GDAL 3.3, not 3.1: https://github.com/OSGeo/gdal/issues/4467#issuecomment-994838301. I was getting build issues with newer setuptools and GDAL 3.2.

Fixed in #29496